ops/terraform/services/base/values/test.yaml

```diff
@@ -152,6 +152,8 @@
 /bfd/${env}/pipeline/nonsensitive/rda/cleanup/run_size: UNDEFINED
 /bfd/${env}/pipeline/nonsensitive/rda/cleanup/transaction_size: UNDEFINED
 /bfd/${env}/pipeline/nonsensitive/rda/instance_type: m6a.large
+/bfd/${env}/pipeline/nonsensitive/rda/samhsa/backfill/enabled: "false"
+/bfd/${env}/pipeline/nonsensitive/rda/samhsa/backfill/batch_size: 15000  
```

Comment:
Might want to test with a larger batch size here to see if it helps. 100,000 or so shouldn't be a problem.